Clonamos el repositorio en ambas carpetas.

Hacemos un cambio en casa y otro en instituto.

Subimos le cambio en casa.

Sin hacer pull subimos el cambio en instituto y nos da este error:

To https://github.com/migangal4/PracticaErrores.git
 ! [rejected]        master -> master (fetch first)
error: failed to push some refs to 'https://github.com/migangal4/PracticaErrores
.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Solucionamos esto haciendo "pull" antes de subir lo de instituto

Volvemos a hacer cambios y volvemos a subirlos de la misma manera, dandonos el mismo error. Lo solucionamos haciendo "pull" de nuevo

Una vez mas hacemos los cambios que nos pide el enunciado y una vez mas nos da un error por no haber hecho "pull" antes de subir los cambios

Volvemos a solucionarlo haciendo pull antes de aplicar y subir los cambios para que la carpeta este en la ultima version al subir el cambio realizado